samples/django/django_user/myoauth/urls.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2016 Google Inc.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from django.conf.urls import include, url
+from django.contrib import admin
+import django.contrib.auth.views
```

Comment:
Ah was reading down and now I see you mentioning this. If you feel strongly about sticking to `oauth2client` style we can. Following type conventions is obviously more important than following import conventions. I don't think it's super important to demonstrate Django-compliant style, and I don't even necessarily think importing methods is a strict convention although Django does do it in its tutorials. In any case given this is clearly not part of the `oauth2client` package and is a Django sample my inclination is to follow Django conventions but in places where it really can go either way like import conventions, don't feel strongly about it and can change it.